doc/source/tutorials/lab.rst

```diff
@@ -396,19 +400,25 @@ should be a subset of :class:`qibolab.qubits.Qubit` attributes.
 Providing the above runcard is not sufficient to instantiate a
 :class:`qibolab.platform.Platform`. This should still be done using a
 ``create()`` method, however this is significantly simplified by
-``qibolab.serialize``. Here is the ``create()`` method that loads the parameters of
+``qibolab.serialize``. The ``create()`` method should be put
+Here is the ``create()`` method that loads the parameters of
```

Comment:
```suggestion
``qibolab.serialize``.
Here is the ``create()`` method that loads the parameters of
```

Reply:
Forgot to complete the sentence, thanks.